### DISCUSSION TITLE
Oqtane emails triggering Office 365 spam detection and quarantined

### DISCUSSION THREAD

**Discussion by enfJoao** [oqtane/oqtane.framework#4843] [General]
I have used this email account in smtp configuration for other CMS and never had problems.

Somehow, the emails send by Oqtane (2FA, SMTP test) are triggering Office 365 spam protection and placed in quarantine. This is not the usual spam that you retrive with one click. It's the quarantine where you receive a notification at the day's end informing you have quarantined items.

Ther's no specific information on why this happens, but it seems to be related to the email header.

**Comment by sbwalker** [comment]
@enfJoao Oqtane does not add any custom email headers... however it does send all emails in Html format.

**Comment by enfJoao** [comment]
The email itself has several issues on the code that generates it, but the outlook quarantine detection must be caused by something else. I never had issues with DNN emails, and they are also html constructed.

![edited_Screenshot_20241118_192822_Outlook](https://github.com/user-attachments/assets/a6079880-d1c2-4883-969d-1c5d5ff06e27)

**Comment by sbwalker** [comment]
It looks like an issue was introduced in this commit where email notifications were localized:

#3794

**Comment by sbwalker** [comment]
the email template issue has been fixed in #4845
